app/components/verkiezingen/prepare-legislatuur-section.hbs

```diff
@@ -0,0 +1,54 @@
+{{#if (await this.isBurgemeester)}}
+  <Mandaat::BurgemeesterSelector @bestuursorgaanInTijd={{@bestuursorgaan}} />
+{{else}}
+  <AuToolbar
+    @size="large"
+    @nowrap="true"
+    class="au-u-padding-bottom-none au-u-margin-bottom-none"
+    as |Group|
+  >
+    <Group class="au-u-margin-bottom-none">
+      <div class="au-o-flow">
+        <AuHeading @skin="3">Samenstelling
+          {{@bestuursorgaan.isTijdsspecialisatieVan.naam}}</AuHeading>
+      </div>
+    </Group>
+  </AuToolbar>
+  <Verkiezingen::DraftMandatarisList
```

Comment:
The sorting is indeed a little funcky ;p the calls are happening but in the UI i don't see anything happing

Reply:
sorting can wait for a separate pr, we need this first